### ISSUE TITLE
`@stylistic/function-parentheses-newline-inside` declines an opening fix where the argument already stands inside the comment

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#281]
Found while fixing #132, by checking whether the rewriting that issue asks for belongs on the opening parenthesis as well. It does. The guard #129 put there asks one question where the closing guard asks two, and the one it leaves out is the one that grants a fix: it never asks whether a comment holds the first argument already.

### What minimal example or steps are needed to reproduce the bug?

```less
a {
  t: foo(1px // c) calc( 2px
);
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "postcss-less",
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
    "@stylistic/function-parentheses-newline-inside": "never-multi-line"
  }
}
```

### What did you expect to happen?

The space after `calc(` taken away. The `// c) calc( 2px` comment runs to the line break behind `2px`, so the first argument stands inside it before the fix and inside it afterwards; emptying that space moves nothing anywhere it was not.

### What actually happened?

The problem is reported and nothing is written, on every run.

The guard is at [`lib/rules/function-parentheses-newline-inside/index.js:165`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/function-parentheses-newline-inside/index.js#L165):

```js
let isOpeningFixable = !endsWithInlineComment(declValue.slice(0, firstIndex), spellsInlineComments)
```

`endsWithInlineComment` trims the trailing whitespace away, so what this reads is the text the fix would leave behind — the second of the two questions the closing guard asks. The first, whether a comment holds the argument as the value stands, is never put, so a value already holding the argument inside a comment is declined along with one the fix would carry in.

This is the same family as #132 on the opening side: a fix refused where nothing is at risk. It is not the same defect, though — #132 was two questions read across two readings of a line break, and this is one question standing where two belong.

### Do you have a proposal to fix the bug?

Ask both questions and decline only where the argument goes from outside a comment to inside one, as the closing guard does since #132:

```
unsafe = not inside_before and inside_after
```

`beforeClosingString` is gone from that rule; `movesClosingIntoComment` is what it became, and the opening side needs the same shape read from the value tree its own way. Both askings should be put to each reading of a line break, the breaks both syntaxes read and those with a form feed among them, which is what the closing guard does now.

`function-parentheses-space-inside` carries no opening guard at all, and #280 is that half of this.

Whether the opening guard should read the copy the syntax prints, the way the closing one does through `printedBefore`, is a question of its own: it reads `declValue` directly today.
